bigbluebutton-html5/imports/api/users/server/modifiers/addUser.js

```diff
@@ -73,6 +73,7 @@ export default function addUser(meetingId, userData) {
         inactivityCheck: false,
         responseDelay: 0,
         loggedOut: false,
+        whiteboardAccess: user.presenter,
```

Comment:
What happens when a user joins a meeting when the presenter set the multiuser mode? Would the whiteboard toolbar be displayed?

Reply:
@hiroshisuga  At the moment, if a user joins after the presenter sets the multiuser mode the toolbar isn't displayed. It's behaving similar to the polling where users who join after initiation are excluded.

Reply:
@KDSBrowne it's fine so far. But I prefer it displayed depending on the multiuser mode (whiteboardAccess : user.presenter || multiuser). Students sometimes want to rejoin the meeting due to a network failure. When the teacher lets all the students work on the whiteboard, it would be very difficult to realize that this rejoining student does not have the access. On the other hand, when only a few students have access to the whiteboard, the teacher should realize (by the number on the toolbar) that the rejoining student also have an access and the teacher can remove it if necessary.